lib/contracts.ts

```diff
@@ -366,7 +366,7 @@ export class ReadOnlyOptionPair implements IReadOptionPair {
         const [expiryTime, windowSize, strikePrice] = await Promise.all([
             this.option.expiryTime(),
             this.option.windowSize(),
-            this.option.strikePrice(),
+            this.obligation.strikePrice(),
```

Comment:
What is the purpose of that change? Not sure I follow why we draw one property from the obligation and the others from the option.

Reply:
We no longer store the `strikePrice` in the option contract, hence we need to read from the obligation contract.

Reply:
Ah, right OK. Should we store the strike price in the contract though? So you could query it from both contracts individually without having to make a call?

Reply:
Since this is read-only we don't incur any fees for fetching this here, however if we additionally add it to the option contract it would increase storage costs.

Reply:
Ok, then we leave at is!